alab_management/sample_view/sample.py

```diff
@@ -22,8 +22,8 @@ class Sample:
     task_id: Optional[ObjectId]
     name: str
     position: Optional[str]
-    metadata: Optional[Dict[str, Any]] = field(default_factory=dict)
-    tags: Optional[List[str]] = field(default_factory=list)
+    metadata: Optional[Dict[str, Any]] = field(default_factory=lambda: None)
```

Comment:
Why cannot it default to list/dict? That should be easier for searching in the MongoDB.